apps/andi/lib/andi_web/live/edit_live_view.ex

```diff
@@ -216,11 +220,42 @@ defmodule AndiWeb.EditLiveView do
     {:noreply, assign(socket, test_results: results, testing: false)}
   end
 
+  def handle_info({:validate, %{"form_data" => form_data}}, socket), do: handle_validation(form_data, socket)
+
+  def handle_info({:add_key_value, %{"field" => field}}, socket) do
```

Comment:
It would be nice if it were more clear where this message is coming from. We figured it out, as it was coming from the Key Value Editor

Reply:
The "add_key_value" message is sent from the event handler in the KeyValueEditor... this matching string should make it relatively clear how these are linked.  Will continue to think about how to make this better.